public/privacy.html

```diff
@@ -0,0 +1,422 @@
+        <div class="footer-top-left"><img src="images/Logo.svg" loading="lazy" alt="" class="brand footer-brand">
+          <p class="footer-text">An open-source release orchestration &amp; CI/CD platform for Kubernetes</p>
+          <a href="https://github.com/devtron-labs/devtron" target="_blank" class="nav-button footer w-inline-block gitshow"><img src="images/Github-logo.svg" loading="lazy" alt="" class="nav-icon">
+            <div>View on Github</div>
+          </a>
+        </div>
+        <div class="footer-top-right">
+          <div class="social-grid">
+            <a href="https://www.linkedin.com/company/devtron-labs/" target="_blank" class="social-wrapper w-inline-block"><img src="images/Insta.svg" loading="lazy" alt=""></a>
+            <a href="https://twitter.com/DevtronL" target="_blank" class="social-wrapper w-inline-block"><img src="images/Twitter.svg" loading="lazy" alt=""></a>
+            <a href="https://www.youtube.com/channel/UCAHRp9qp0z1y9MMtQlcFtcw/featured" target="_blank" class="social-wrapper w-inline-block"><img src="images/Youtube.svg" loading="lazy" alt=""></a>
+            <a href="https://www.facebook.com/devtronlabs" target="_blank" class="social-wrapper w-inline-block"><img src="images/Facebook.svg" loading="lazy" alt=""></a>
+          </div>
+          <div class="footer-card"><img src="images/Footer-image.jpg" loading="lazy" sizes="(max-width: 479px) 93vw, (max-width: 991px) 293.421875px, 328.9921875px" srcset="images/Footer-image-p-500.jpeg 500w, images/Footer-image.jpg 513w" alt="" class="footer-image">
+            <a href="https://www.facebook.com/groups/353685209144241/?ref=group_header" target="_blank" class="footer-card-content w-inline-block">
+              <div>
+                <div class="card-title-caps">Join the community</div>
+                <p class="small-text bold">DevOps &amp; Kubernetes Professionals</p>
+              </div><img src="images/Facebook.svg" loading="lazy" alt="" class="card-icon">
+            </a>
+          </div>
+        </div>
+      </div>
+      <div class="footer-bottom white _50--opacity">
+        <div class="flex">
+            <div class="mr-36">© 2022 DevtronLabs. All rights reserved</div>
+            <a class="white tcn" href="privacy.html">Privacy policy</a>
+          </div>
+        <div>Made with <span class="text-span">♥</span> in India</div>
+      </div>
+    </div>
+  </div>
+  <script src="https://d3e54v103j8qbb.cloudfront.net/js/jquery-3.5.1.min.dc5e7f18c8.js?site=61aeec2bcab0f63538b9256e" type="text/javascript" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
+  <script src="js/webflow.js" type="text/javascript"></script>
+  <!-- [if lte IE 9]><script src="https://cdnjs.cloudflare.com/ajax/libs/placeholders/3.0.2/placeholders.min.js"></script><![endif] -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/1.7.1/clipboard.min.js"></script>
+  <script>
```

Comment:
We can remove this unnecessary script

Reply:
This script is necessary for hamburger menu to work